src/components/suggest/suggest_item.tsx

```diff
@@ -81,6 +91,16 @@ interface ColorToClassMap {
   [key: string]: string;
 }
 
+export type LabelWidthSize =
```

Comment:
No need to export this unless you are reusing in another file.
```suggestion
type LabelWidthSize =
```